cancellation/src/starter/index.ts

```diff
@@ -9,7 +9,7 @@ async function run() {
 
   await workflow.start();
 
-  await new Promise(resolve => setTimeout(resolve, 100));
+  await new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
In 2 years you'll be able to use this: https://nodejs.org/api/timers.html#timers_timerspromises_settimeout_delay_value_options

Reply:
That will be excellent, thanks for the heads up.